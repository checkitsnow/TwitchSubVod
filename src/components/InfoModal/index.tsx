import React from 'react';
import { FiInfo } from 'react-icons/fi';

import { Container } from './styles';

const InfoModal: React.FC = () => {
  return (
    <Container>
      <div className="info-box">
        <p>
          You can get the Vod Id from
          <a
            href="https://twitchtracker.com"
            target="_blank"
            rel="noopener noreferrer"
          >
            twitchtracker.com
          </a>
        </p>
      </div>
      <FiInfo size={18} />
    </Container>
  );
};

export default InfoModal;
