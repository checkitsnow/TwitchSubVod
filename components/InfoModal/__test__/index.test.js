import React from 'react';
import renderer from 'react-test-renderer';
import InfoModal from '..';

it('should render Info Modal correctly', () => {
  const tree = renderer.create(<InfoModal />).toJSON();
  expect(tree).toMatchSnapshot();
});
