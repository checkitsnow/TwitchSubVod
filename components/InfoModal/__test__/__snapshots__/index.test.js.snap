// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Info Modal correctly 1`] = `
<div
  className="styles__Container-pgyzn6-0 kCeIyN"
>
  <div
    className="info-box"
  >
    <p>
      You can get the Vod Id from
      <a
        href="https://twitchtracker.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        twitchtracker.com
      </a>
    </p>
  </div>
  <svg
    fill="none"
    height={18}
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 24 24"
    width={18}
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="12"
      cy="12"
      r="10"
    />
    <line
      x1="12"
      x2="12"
      y1="16"
      y2="12"
    />
    <line
      x1="12"
      x2="12.01"
      y1="8"
      y2="8"
    />
  </svg>
</div>
`;
